src/Umbraco.Infrastructure/PropertyEditors/BlockEditorPropertyEditor.cs

```diff
@@ -264,7 +263,7 @@ public IEnumerable<ValidationResult> Validate(object value, string valueType, ob
                         _textService.Localize("validation", "entriesShort", new[]
                         {
                             validationLimit.Min.ToString(),
-                            (validationLimit.Min - blockEditorData.Layout.Count()).ToString()
+                            (validationLimit.Min - (blockEditorData?.Layout.Count() ?? 0)).ToString()
```

Comment:
I guess it potential need the same null check for `Max` as well here:
https://github.com/umbraco/Umbraco-CMS/pull/11184/files#diff-08b637d7d3f18c3bd184b2d3ebd9563ab2f72b70fde8aa34432404cb3185a99aR277

Reply:
`Max` already has a null check for `blockEditorData` in [Line 271](https://github.com/umbraco/Umbraco-CMS/pull/11184/files#diff-08b637d7d3f18c3bd184b2d3ebd9563ab2f72b70fde8aa34432404cb3185a99aR271), so it shouldn't be needed.

Reply:
Yes, it has a null check for `blockEditorData`:
https://github.com/umbraco/Umbraco-CMS/blob/f421fa72935aaa1dbcc5dbf7fc3926f21b1c2fc2/src/Umbraco.Infrastructure/PropertyEditors/BlockEditorPropertyEditor.cs#L271

but shouldn't this always a null for `blockEditorData` as well?
https://github.com/umbraco/Umbraco-CMS/blob/f421fa72935aaa1dbcc5dbf7fc3926f21b1c2fc2/src/Umbraco.Infrastructure/PropertyEditors/BlockEditorPropertyEditor.cs#L259-L269

Reply:
Well, in my case I had no items so it was ending up `null` and causing the initial issue, so if it's returning `null` when there are no items I think it makes sense to show the entries short validation message.